### PR TITLE
Add program GovermentElections

### PR DESCRIPTION
**Pull request by sarafuentes3025**

### CODE REVIEW

File: src/main/java/com/sfuentes/elections/Result.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      }
+    }
+    candidates.put("Mauricio: ", firstCandidateCount);
+    candidates.put("Joaquin: ", secondCandidateCount);
+    candidates.put("Rodrigo: ", thirdCandidateCount);
+
+    return candidates;
+  }
+
+  public Map<String, Integer> getWinningCandidate() {
```
Comment (aandraca) [inline src/main/java/com/sfuentes/elections/Result.java:35]: This should be a class like ElectionResults.

**Review by aandraca — CHANGES_REQUESTED** [review]

File: src/test/java/com/sfuentes/electionsTest/ResultTest.java [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+public class ResultTest {
+
+  @Test
+  void getWinningCandidate_allCandidates_rodrigoIsTheWinner() {
+    Vote vote1 = new Vote("mauricio");
+    Vote vote2 = new Vote("rodrigo");
+    Vote vote3 = new Vote("rodrigo");
+    Vote vote4 = new Vote("joaquin");
+    List<Vote> votes = List.of(vote1, vote2, vote3, vote4);
```
[the remark this answers is not in the document]
Comment (sarafuentes3025) [reply src/test/java/com/sfuentes/electionsTest/ResultTest.java]: Fixed

File: src/main/java/com/sfuentes/elections/Result.java (hunk 1 above)
Comment (sarafuentes3025) [reply src/main/java/com/sfuentes/elections/Result.java]: Fixed

**Review by aandraca — APPROVED** [review]
LGTM

**Pull request merged**